sfm/ui/views.py

```diff
@@ -196,10 +197,26 @@ def get_context_data(self, **kwargs):
                                                     harvest__historical_collection__id=self.object.id).exists()
         context["item_id"] = self.object.id
         context["model_name"] = "collection"
-
         context["status_choices"] = Harvest.STATUS_CHOICES
         return context
 
+def download(request, pk):
```

Comment:
Can you make the method name slightly more descriptive, like `download_seed_list`?